aws-py-dynamicresource/__main__.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+import json
+import base64
+import pulumi
+import pulumi_aws as aws
+import pulumi_mysql as mysql
+from MySqlDynamicProvider import *
+
+# Get neccessary settings from the pulumi config
+config = pulumi.Config()
+admin_name = config.require("sql-admin-name")
+admin_password = config.require_secret("sql-admin-password")
+user_name = config.require("sql-user-name")
+user_password = config.require_secret("sql-user-password")
+availability_zone = pulumi.Config("aws").get("region")
+
+# Creating a VPC and a public subnet
+app_vpc = aws.ec2.Vpc("app-vpc",
+    cidr_block="172.31.0.0/16",
+    enable_dns_hostnames=True)
+
+app_vpc_subnet = aws.ec2.Subnet("app-vpc-subnet",
+    cidr_block="172.31.0.0/20",
+    availability_zone=availability_zone + "a",
+    vpc_id=app_vpc)
+
+# Creating a gateway to the web for the VPC
+app_gateway = aws.ec2.InternetGateway("app-gateway",
+    vpc_id=app_vpc.id)
+
+app_routetable = aws.ec2.RouteTable("app-routetable",
+    routes=[
+        {
+            "cidr_block": "0.0.0.0/0",
+            "gateway_id": app_gateway.id,
+        }
+    ],
```

Comment:
`pulumi_aws` 3.2.1 now supports classes for nested data. I've been working on updating all our existing Python examples to use them, but we should also use them for all new Python examples going forward.

```suggestion
    routes=[
        aws.ec2.RouteTableRouteArgs(
            cidr_block="0.0.0.0/0",
            gateway_id=app_gateway.id,
        )
    ],
```